lib/configDefaults.js

```diff
@@ -1,7 +1,11 @@
 const os = require('os')
 const path = require('path')
 
-module.exports = {
+const reporter = require('./reporter')
+
+const configFile = path.join(os.homedir(), '.d2rc')
```

Comment:
Let's be opinionated and use `.config/d2/(d2.)?config.js(on)?`

Reply:
Done in 3e55cf1.